2006Scape Client/src/main/java/ClientSettings.java

```diff
@@ -9,9 +9,39 @@
 public class ClientSettings {
     /**
      * @QoL
-     * Enables Server-Wide Snow In The Designated Month
+     * Hide roofs
      */
-    public static String SNOW_MONTH = ".12";
+    public static boolean HIDE_ROOFS = true;
+    /**
+     * @QoL
+     * Forces Server-Wide Snow floor
+     */
+    public static boolean SNOW_FLOOR_FORCE_ENABLED = false;
+    /**
+     * @QoL
+     * Toggles Server-Wide Snow floor
+     */
+    public static boolean SNOW_FLOOR_ENABLED = false;
+    /**
+     * @QoL
+     * Enables Server-Wide Snow floor In The Designated Month
+     */
+    public static int SNOW_FLOOR_MONTH = 12;
+    /**
+     * @QoL
+     * Forces Server-Wide Snow overlay
+     */
+    public static boolean SNOW_OVERLAY_FORCE_ENABLED = false;
+    /**
+     * @QoL
+     * Toggles Server-Wide Snow overlay
+     */
+    public static boolean SNOW_OVERLAY_ENABLED = false;
+    /**
+     * @QoL
+     * Enables Server-Wide Snow overlay In The Designated Month
+     */
+    public static String SNOW_OVERLAY_MONTH = ".12";
```

Comment:
This and line 29 I think should be combine into one var? It makes sense to enable both the overlay and the floor snow effects in December, right?

Reply:
We could combine the snow month variables into one variable, but, I disagree that the floor and overlay should be the same toggle varriable. They are two distinctly different features. Some players may want only the snow feature whereas others may want only the overlay feature, whereas others will want both - but not all would want both.

I think having more customization is a good thing.

Reply:
I changed it so there's only one snow month variable as requested so I'll mark this as resolved.